api/src/main/proto/stellarstation/api/v1/groundstation/groundstation.proto

```diff
@@ -119,16 +119,18 @@ message Plan {
   // need to configure transmission during the plan to match this device.
   radio.RadioDeviceConfiguration uplink_radio_device = 6;
 
-  // The start time of the plan.
+  // The start of the time window reserved for the plan.
   //
-  // This value will never be after aos_time. Unlike aos_time, this value is fixed upon plan
-  // creation.
+  // This timestamp is always before aos_time and should be used by the ground station to prepare
+  // for the upcoming plan. Unlike aos_time, this timestamp does not change after the plan is
```

Comment:
We use ` to surround names of fields in comments. I don't think it actually does anything in documentation, etc but it's good to mark out field names somehow.